src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Interop.cs

```diff
@@ -30,11 +26,37 @@ public struct PROCESS_MEMORY_COUNTERS
         [DllImport("api-ms-win-core-memory-l1-1-1.dll")]
         public static extern bool GetProcessWorkingSetSizeEx(SafeProcessHandle hProcess, out IntPtr lpMinimumWorkingSetSize, out IntPtr lpMaximumWorkingSetSize, out uint flags);
 
-
         [DllImport("api-ms-win-core-processthreads-l1-1-0", CharSet = System.Runtime.InteropServices.CharSet.Unicode, SetLastError = true)]
         public static extern int GetPriorityClass(SafeProcessHandle handle);
 
         [DllImport("kernel32.dll", CharSet = System.Runtime.InteropServices.CharSet.Ansi, SetLastError = true)]
         public static extern SafeProcessHandle GetCurrentProcess();
+
+        [DllImport("api-ms-win-core-processthreads-l1-1-0.dll")]
+        internal static extern int GetCurrentProcessId();
+
+        [DllImport("libc")]
```

Comment:
There are not a lot of interop to do for tests, and having a file for each platform is not worthwhile as the structure is not so complex. Following with what already existed here.